### ISSUE TITLE
circle-ci error: node version needs to upgraded from 9.4.0 since jest need minimum node v10.13.0

### ISSUE THREAD

**Issue by hashtd** [hawkeye64/onvif-nvt#436]
https://app.circleci.com/pipelines/github/hawkeye64/onvif-nvt/326/workflows/609fc4c4-5e7d-4a26-9a73-1ed2ba4ea6c1/jobs/1070

```
onvif-nvt@0.3.0 test
jest

/home/circleci/onvif-nvt/node_modules/jest-cli/build/cli/index.js:227
    } catch {
            ^

SyntaxError: Unexpected token {
    at new Script (vm.js:51:7)
    at createScript (vm.js:138:10)
    at Object.runInThisContext (vm.js:199:10)
    at Module._compile (module.js:624:28)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)

Exited with code exit status 1

CircleCI received exit code 1
```

**Comment by hawkeye64** [comment]
@hashtd I am not sure why you would get that error when in package.json:
```
  "engines": {
    "node": ">=8.0.0"
  },

```

**Comment by hawkeye64** [comment]
Fix will be in v0.3.1
